src/components/WYSIWYG/WYSIWYG.tsx

```diff
@@ -46,19 +49,20 @@ export const WYSIWYG: FunctionComponent<WYSIWYGProps> = ({
 	svgIconsPath,
 	btnsDef,
 	plugins,
-	onFocus,
-	onBlur,
-	onInit,
-	onChange,
-	onResize,
-	onPaste,
-	onOpenFullScreen,
-	onCloseFullScreen,
-	onClose,
+	onFocus = () => {},
+	onBlur = () => {},
+	onInit = () => {},
+	onChange = () => {},
+	onResize = () => {},
+	onPaste = () => {},
+	onOpenFullScreen = () => {},
+	onCloseFullScreen = () => {},
+	onClose = () => {},
 }: WYSIWYGProps) => (
 	<Trumbowyg
 		id={id}
 		data={data}
+		placeholder={placeholder}
```

Comment:
add a placeholder to the second story: 
![image](https://user-images.githubusercontent.com/1710840/63411280-8195e900-c3f5-11e9-9b4c-8a48212ca061.png)